### ISSUE TITLE
Underline added with link causing UI issue

### ISSUE THREAD

**Issue by Prakash-karena** [Aymkdn/html-to-pdfmake#108]
Here on first google i have added only underline which was working fine. but on second google text fist i have added link and after that added underline which shows underline after the text as well like below image. 

Html content is like below:

`<p>Only underline added in this text :-&nbsp;<u>&nbsp;google</u>&nbsp;&nbsp;</p>↵↵<p>First added underline and after that added link into it :-&nbsp;&nbsp;<u><a href="https://google.com">google</a></u></p>↵↵<p>&nbsp;</p>`

![image](https://user-images.githubusercontent.com/63772512/123201259-ea59bb00-d4cf-11eb-9cda-54fc59039724.png)

Here is generated code and pdf 
[pdf-underline-issue.pdf](https://github.com/Aymkdn/html-to-pdfmake/files/6706170/pdf-underline-issue.pdf)

```js
// http://pdfmake.org/playground.html
var dd = {
	content: 
     
		    [{
		"text": [{
				"text": "Only underline added in this text :- ",
				"margin": [
					0,
					5,
					0,
					10
				],
				"style": [
					"html-p"
				]
			},
			{
				"text": " only underline",
				"nodeName": "U",
				"decoration": [
					"underline"
				],
				"style": [
					"html-u",
					"html-p"
				]
			},
			{
				"text": "  ",
				"margin": [
					0,
					5,
					0,
					10
				],
				"style": [
					"html-p"
				]
			}
		],
		"nodeName": "P",
		"margin": [
			0,
			5,
			0,
			10
		],
		"style": [
			"html-p"
		],
		"_margin": [
			0,
			5,
			0,
			10
		],
		"_inlines": [],
		"_minWidth": 49.224609375,
		"_maxWidth": 259.412109375,
		"positions": [{
			"pageNumber": 1,
			"pageOrientation": "portrait",
			"pageInnerHeight": 666.89,
			"pageInnerWidth": 495.28,
			"left": 55,
			"top": 453.75,
			"verticalRatio": 0.515452323471637,
			"horizontalRatio": 0.010095299628492975
		}]
	},
	{
		"text": [{
				"text": "First added underline and after that added link into it :-  ",
				"margin": [
					0,
					5,
					0,
					10
				],
				"style": [
					"html-p"
				]
			},
			{
				"text": [{
					"text": "underline with link",
					"decoration": [
						"underline",
						"underline"
					],
					"color": "blue",
					"style": [
						"html-a",
						"html-u",
						"html-p"
					],
					"link": "https://google.com",
					"nodeName": "A"
				}],
				"nodeName": "U",
				"decoration": [
					"underline"
				],
				"style": [
					"html-u",
					"html-p"
				]
			}
		],
		"nodeName": "P",
		"margin": [
			0,
			5,
			0,
			10
		],
		"style": [
			"html-p"
		],
		"_margin": [
			0,
			5,
			0,
			10
		],
		"_inlines": [],
		"_minWidth": 49.224609375,
		"_maxWidth": 388.79296875,
		"positions": [{
			"pageNumber": 1,
			"pageOrientation": "portrait",
			"pageInnerHeight": 666.89,
			"pageInnerWidth": 495.28,
			"left": 55,
			"top": 482.8125,
			"verticalRatio": 0.5590314744560573,
			"horizontalRatio": 0.010095299628492975
		}]
	},
	{
		"text": " ",
		"nodeName": "P",
		"margin": [
			0,
			5,
			0,
			10
		],
		"style": [
			"html-p"
		],
		"_margin": [
			0,
			5,
			0,
			10
		],
		"_inlines": [],
		"_minWidth": 0,
		"_maxWidth": 0,
		"positions": [{
			"pageNumber": 1,
			"pageOrientation": "portrait",
			"pageInnerHeight": 666.89,
			"pageInnerWidth": 495.28,
			"left": 55,
			"top": 511.875,
			"verticalRatio": 0.6026106254404775,
			"horizontalRatio": 0.010095299628492975
		}]
	},
      ]
}
```

**Comment by Aymkdn** [comment]
I've just published v2.2.6 that should fix this behavior.

**Comment by Prakash-karena** [comment]
Thank you @Aymkdn now working fine.
